packages/console/src/pages/ApiResources/index.tsx

```diff
@@ -72,9 +74,15 @@ const ApiResources = () => {
             </tr>
           )}
           {data?.map(({ id, name, indicator }) => (
-            <tr key={id} className={styles.clickable}>
+            <tr
+              key={id}
+              className={styles.clickable}
+              onClick={() => {
+                navigate(buildDetailsLink(id));
```

Comment:
If we add onClick handler on `tr`, we don't need to handle that navigation in the next `td`

Reply:
per offline [discussion](https://silverhand-io.slack.com/archives/C02A8G4HVAM/p1646798668637219), we allow the user  right-click the title and  open the details in a new tab or window.
@demonzoo

Reply:
ok, got it